co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureMessageAmbassador.java

```diff
@@ -47,21 +46,16 @@ public class InfrastructureMessageAmbassador extends AbstractFederateAmbassador
      */
     InfrastructureConfiguration infrastructureConfiguration;
 
-    /**
-     * List of infrastructures that are controlled
-     */
-    private final HashMap<String, Boolean> v2xMap = new HashMap<>();
-
     /**
      * The number of CARMA vehicles.
      */
     int numberOfInfrastructureInstances = 0;
```

Comment:
Not part of your changeset, but this should almost certainly be private. And the comment above references "CARMA Vehicles" should be changed to infrastructure

Reply:
I believe this one can be removed since the list of instances can be accessed by using the function Eric implemented `getManagedInstances`

Reply:
Yeah, I think i'd be okay with removing it if you want to go ahead and delete it in this PR, provided it's not used anywhere else.